src/components/Questionnaire/QuestionTypes/DiagnosisQuestion.tsx

```
@@ -14,6 +14,7 @@
   useState,
 } from "react";
 import { useTranslation } from "react-i18next";
+import { useTranslation } from "react-i18next";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'useTranslation'. Consider to delete it or rename it.

'useTranslation' is defined here:


(lint/suspicious/noRedeclare)